### PR TITLE
Dev to Master

### PR DESCRIPTION
**Pull request by rishi-shah12**

### CODE REVIEW

File: variantEffectPredictor.wdl [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    File? targetBed
+    File? custom
+    Boolean? toMAF
+    String tumorName
+    String normalName
+  }
+
+  String customCommand = if defined(custom) == true then "true" else "false"
+
+  if (defined(targetBed) == true) {
```
Comment (mlaszloffy) [inline variantEffectPredictor.wdl:10]: Can the task calls be collapsed into:
1) if defined(targetBed) call targetBedTask
2) call vep {      input: 
vcfFile = select_first([targetBedTask.vcfFile,vcfFile],
vcfIndex = select_first([targetBedTask.vcfIndex, vcfIndex]),
...
3) if toMAF call vcf2maf

And then outputs modified to:
 File outputVcf = vep.vepVcfOutput
 File? outputMaf = vcf2maf.mafOutput

**Review by mlaszloffy — CHANGES_REQUESTED** [review]

File: variantEffectPredictor.wdl [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+           vepCacheModule = vepCacheModule,
+           vepCacheDir = vepCacheDir,
+           pluginModule = pluginModule, 
+           pluginDir = pluginDir
+    }
+  
+  if (toMAF == true) {
+    if (onlyTumor == true) {
+      call tumorOnlyAlign {
+      input: vcfFile = vep.vepVcfOutput,
```
Comment (prisnir-zz) [inline variantEffectPredictor.wdl]: The input for this should be input VCF and not the VEP annotated VCF

Comment (prisnir-zz) [reply variantEffectPredictor.wdl]: Looks like:
vcf2maf is enabled for tumor only VCFs
and it reads VEP annotated VCF to run vcf2maf

File: tests.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+},
+{
+    "id": "vepWorkflow_MAF",
+    "parameters": {
+      "wdl_inputs": {
+         "variantEffectPredictor.vcfFile": "/.mounts/labs/gsi/testdata/variantEffectPredictor/input_data/IMP-01710-ARC1.mutect.strelka.snv.indel.vcf.gz",
+         "variantEffectPredictor.onlyTumor": true,
+         "variantEffectPredictor.toMAF": true,
+         "variantEffectPredictor.vepCacheModule": "vep-hg19-cache/92",
+         "variantEffectPredictor.vepCacheDir": "$VEP_HG19_CACHE_ROOT/.vep",
```
Comment (prisnir-zz) [inline tests.json]: Can we also include a test case for GRCh38 ?

**Review by prisnir-zz — CHANGES_REQUESTED** [review]

**Comment by xuemeiluo** [comment]
retest this please

**Review by mlaszloffy — APPROVED** [review]

**Review by prisnir-zz — APPROVED** [review]

**Pull request merged**